From 77af679c6496f933980ce6280858317e6d3f7e6d Mon Sep 17 00:00:00 2001
From: Johnny <jonnyhihi@gmail.com>
Date: Tue, 27 Oct 2015 12:28:40 +0800
Subject: [PATCH 2/2] Modify channel
 OPEN_POWER_N_PLUS_ONE_BULK_POWER_LIMIT_WATTS=1200 and ADC_CHANNEL_OFFSET=0

                        ADC_CHANNEL_ID  ADC_CHANNEL_ASSIGNMENT  ADC_CHANNEL_GAIN        ADC_CHANNEL_OFFSET
Memory_Proc0_Power      1               6                       10000                   0
Memory_Proc1_Power      2
Memory_Proc2_Power      3
Memory_Proc3_Power      4
Proc0_Power             5               7                       10000                   0
Proc1_Power             6               3                       10000                   0
Proc2_Power             7               1                       10000                   0
Proc3_Power             8               0                       10000                   0
PCIE_Proc0_Power        9               13                      10000                   0
PCIE_Proc1_Power        10
PCIE_Proc2_Power        11
PCIE_Proc3_Power        12
IO_A_Power              13              14                      10000                   0
IO_B_Power              14
IO_C_Power              15
Fan_Power_A             16
Fan_Power_B             17
Storage_Power_A         18              15                      10000                   0
Storage_Power_B         19
Total_System_Power      22
Memory_Cache_Power      23
Memory_Proc0_0_Power    25              5                       10000                   0
Memory_Proc0_1_Power    26              4                       10000                   0
Memory_Proc0_2_Power    27              2                       10000                   0
12V_Sense               20              9                       6500                    0
Ground_Sense            21
GPU_Sense               24
                        ADC_CHANNEL_ID  ADC_CHANNEL_ASSIGNMENT  ADC_CHANNEL_GAIN        ADC_CHANNEL_OFFSET
---
 barreleye.xml | 26 +++++++++++++-------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/barreleye.xml b/barreleye.xml
index 023fe02..344478c 100644
--- a/barreleye.xml
+++ b/barreleye.xml
@@ -5837,7 +5837,7 @@
 	</attribute>
 	<attribute>
 		<id>OPEN_POWER_N_PLUS_ONE_BULK_POWER_LIMIT_WATTS</id>
-		<default>4095</default>
+		<default>1200</default>
 	</attribute>
 	<attribute>
 		<id>OPEN_POWER_N_PLUS_ONE_MAX_MEM_POWER_WATTS</id>
@@ -16115,7 +16115,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-31</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -16415,7 +16415,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-32</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -16490,7 +16490,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-30</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -16565,7 +16565,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-29</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -16640,7 +16640,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-35</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -16715,7 +16715,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-37</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17015,7 +17015,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-33</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17390,7 +17390,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-33</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17690,7 +17690,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-32</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17765,7 +17765,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-31</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17840,7 +17840,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-31</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
@@ -17915,7 +17915,7 @@
 	</attribute>
 	<attribute>
 		<id>ADC_CHANNEL_OFFSET</id>
-		<default>-35</default>
+		<default>0</default>
 	</attribute>
 	<attribute>
 		<id>BUS_TYPE</id>
-- 
1.9.1

